docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '2'
+services:
+  ui-tests:
+    build: .
+    environment:
+      REDASH_SERVER_URL: http://redash:5000
+    links:
+      - "server:redash"
+    entrypoint: ./docker-entrypoint.sh
+  server:
+    image: redash/redash:latest
+    command: dev_server
+    depends_on:
+      - postgres
+      - redis
+    ports:
+      - "5000:5000"
+    environment:
+      PYTHONUNBUFFERED: 0
+      REDASH_LOG_LEVEL: "INFO"
+      REDASH_REDIS_URL: "redis://redis:6379/0"
+      REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
+      REDASH_GOOGLE_CLIENT_ID: "dummy"
+      REDASH_GOOGLE_CLIENT_SECRET: "dummy"
+    restart: always
+  worker:
+    image: redash/redash:latest
+    command: scheduler
+    volumes_from:
+      - server
+    depends_on:
+      - server
+    environment:
+      PYTHONUNBUFFERED: 0
+      REDASH_LOG_LEVEL: "INFO"
+      REDASH_REDIS_URL: "redis://redis:6379/0"
+      REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
+      QUEUES: "queries,scheduled_queries,celery"
+      WORKERS_COUNT: 2
+  redis:
+    image: redis:3.0-alpine
+    restart: unless-stopped
+  postgres:
+    image: postgres:9.5.6-alpine
+    # The following turns the DB into less durable, but gains significant performance improvements for the tests run (x3
+    # improvement on my personal machine). We should consider moving this into a dedicated Docker Compose configuration for
+    # tests.
```

Comment:
Let's remove these comments. It will be very confusing for us otherwise at some point in the future, I suppose.